src/core/reducers/allergiesSynopsisReducer.js

```diff
@@ -0,0 +1,32 @@
+import { get } from "lodash";
```

Comment:
all your reducers for custom requests are basically the same. Instead of copy/pasting the code, I suggest you write a reducer creator, which takes the action name as parameter. You'll avoid a lot of duplication.

Reply:
I created function, which creates custom reducer by action name